src/main/java/org/zephyrsoft/sdb2/remote/MQTT.java

```diff
@@ -99,7 +100,7 @@ public void publish(String topic, byte[] payload, int qos, boolean retained) {
 			LOG.debug("Publishing message: {}", topic);
 			try {
 				client.publish(topic, payload, qos, retained);
-				LOG.debug("Payload: " + new String(payload, StandardCharsets.UTF_8));
+				LOG.debug("Payload: " + new String(Arrays.copyOfRange(payload, 0, 50), StandardCharsets.UTF_8));
```

Comment:
What if a file is shorter than 50 bytes? This should not fail just because of logging - or if it does fail, it should give the correct cause (if an exception occurs here, the logged message would be "could not publish message" which is not true).